proto/storage/compliance_operator_v2.proto

```diff
@@ -213,6 +213,7 @@ message ComplianceOperatorScanV2 {
   NodeRole node_selector = 9;
   ScanStatus status = 10;
   google.protobuf.Timestamp created_time = 11;
+  google.protobuf.Timestamp last_started_time = 17; // @gotags: search:"Compliance Scan Last Started Time,hidden"
```

Comment:
A comment to explain the difference between last started time and last executed time would be great.

Reply:
See comment above on numerical order.

Reply:
Done